website/thaliawebsite/api/pagination.py

```diff
@@ -2,12 +2,10 @@
 
 
 class APIv2LimitOffsetPagination(LimitOffsetPagination):
```

Comment:
Je kan hier wel v2 uit de naam halen lijkt me. En ik denk dat er nu misschien ook geen verschil is met de parent class dus misschien can de hele class wel weg?

Reply:
Kan idd vervangen worden